docs/design/tracepoint_definition.md

```diff
@@ -1,30 +1,26 @@
-# トレースポイントの定義
+# Tracepoints definition
 
-CARET はトピック名などの情報を得るための「初期化時の処理に対するトレースポイント」と、  
-測定をするための「ランタイムの処理に対するトレースポイント」があります。
+This section lists all tracepoints and their definition.
+Before listing tracepoints.
 
-本ページではそれぞれのトレースポイントの定義についてまとめています。
+Some tracepoints are used for collecting identification of executors, nodes, callbacks, and topics during application's initialization. They are called initialization tracepoints. The other tracepoints are embedded for sampling timestamps after completion of initialization, and called runtime tracepoints.
 
-各トレースポイントには、その実装方法を区別するために以下の情報も記載しています。
+Almost all of tracepoints supported by CARET are embedded in ROS and DDS layer. CARET utilizes some of the tracepoints embedded in original ROS 2 middleware, which are used for ros2-tracing. Some of the rest tracepoints are added to the fork of ROS 2's rclcpp, the other are introduced by function hooking with LD_PRELOAD. AS explained, tracepoints for CARET is embedded by three ways and they are identified as below.
 
-- Galactic 実装
-  - Galactic 以降の ROS にはデフォルトで実装されている、ros2 tracing が使用しているトレースポイントです。
-  - service や lifecycle など、実装はされていても CARET では使われていないトレースポイントも有ります。
-- CARET フォーク実装
-  - rclcpp リポジトリをフォークして追加しているトレースポイントです。
-- CARET フック実装
-  - LD_PRELOAD によるフックで追加しているトレースポイントです。
+- Original tracepoints
+  - tracepoints embedded in original ROS 2 middleware which are utilized by ros2-tracing
+  - some of tracepoints, for service, action and lifecycle node, are not utilized by CARET
+- Extended tracepoints
+  - CARET-dedicated tracepoints added to the fork of rclcpp
+- Hooked tracepoints
+  - CARET-dedicated tracepoints introduced by function hooking with LD_PRELOAD
 
-> フォークとフックの２つの方法でトレースポイントを追加している理由
->
-> CARET は基本的には実装と切り離すために、LD_PRELOAD を利用したフックによるトレースポイントの追加をしています。  
-> ただし、rclcpp には、プロセス内通信などのテンプレートによる実装を含み、LD_PRELOAD ではフックできない処理があります。  
-> LD_PRELOAD は実行時の動的リンクに対しフックできる機能であるのに対し、  
-> テンプレートで実装された処理はコンパイルでバイナリ内に組み込まれてしまうためフックできません。  
-> このようなテンプレートで実装された処理に対してのみ、rclcpp をフォークしてトレースポイントを追加しています。  
-> このような理由から、測定対象のアプリケーションを再ビルドする必要が生じています。
+<prettier-ignore-start>
+!!! info
+    Please read this section if you are interested in CARET-dedicated tracepoints are extended by the forked rclcpp and LD_PRELOAD. CARET would like to add tracepoints by function hooking as possible. LD_PRELOAD is reasonable to hook functions defined in dynamic library, but it cannot be applied to functions by implemented with C++ template. Such template-based implementation is mapped into binary file after it is built or compiled. Original rclpp uses C++ template for some functions like intra-process communication, for example. The forked rclcpp is introduced to add tracepoints to the functions.
```

Comment:
Original rclpp
Original rcl**c**pp

Reply:
@takam5f2 Could you fix this typo?

Reply:
fixed in https://github.com/tier4/CARET_doc/pull/40/commits/f15fff3805893c68a233edcbfa7d330026cf0265